CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 1.0.4 - 1/15/2016
```

Comment:
This should go into "To be released", the release manager will actually come up with the tag version when we release the new chef recipes.